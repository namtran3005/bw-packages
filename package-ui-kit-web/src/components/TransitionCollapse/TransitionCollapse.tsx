import * as React from 'react';
import { Collapse } from '@material-ui/core';
import { CollapseProps } from '@material-ui/core/Collapse';

const TransitionCollapse: React.FunctionComponent<CollapseProps> = (props) => (
  <Collapse {...props} />
);

export { TransitionCollapse, CollapseProps as TransitionCollapseProps };
