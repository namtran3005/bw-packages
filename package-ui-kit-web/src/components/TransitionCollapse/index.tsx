export {
  TransitionCollapse,
  TransitionCollapseProps,
} from './TransitionCollapse';
