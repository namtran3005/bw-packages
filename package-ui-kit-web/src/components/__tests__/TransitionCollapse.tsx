import * as React from 'react';
import { shallow } from 'enzyme';
import { TransitionCollapse } from '../TransitionCollapse';

describe('TransitionCollapse', () => {
  it('should render TransitionCollapse', () => {
    const wrapper = shallow(<TransitionCollapse />);
    expect(wrapper).toMatchSnapshot();
  });
});
