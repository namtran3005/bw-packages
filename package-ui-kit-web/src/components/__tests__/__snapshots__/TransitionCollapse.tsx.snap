// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransitionCollapse should render TransitionCollapse 1`] = `<WithStyles(ForwardRef(Collapse)) />`;
